src/main/kotlin/se/uulm/snowballr/backend/repository/UserTableRepo.kt

```diff
@@ -170,6 +178,14 @@ class UserTableRepo(
         getEntityByKeyAsResult(::getUserByEmailOrNull, EntityType.USER, email, IdentifierType.EMAIL)
     }
 
+    override suspend fun doesUserExistById(id: UUID): Boolean = db.query {
+        UserTable
+            .select(UserTable.id)
+            .where { UserTable.id eq id }
+            .empty()
+            .not()
```

Comment:
```suggestion
        UserTable.doesEntityExist { UserTable.id eq id }
```
Isn't this possible and shorter?

Reply:
Good point. Didn't recognize that there was a generic helper.

Reply:
There's also `UserTable.doesEntityExistById(id)`, which is even shorter 😄

Reply:
@domienderle If you want to change it, change it to `UserTable.doesEntityExistById(id)`, otherwise this is resolved for me and in any case you can close it on your own.

Reply:
All right. I used `UserTable.doesEntityExistById(id)` now.